app/assets/stylesheets/content/_wiki.sass

```diff
@@ -215,3 +161,12 @@ h1:hover, h2:hover, h3:hover
 
 .wiki-content
   width: 700px
+
+blockquote
+  font-style: italic
```

Comment:
You might want to move the padding applied in `_misc_legacy.sass:350` (right below `filese`) over here too...